modules/darwin/common/packages.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, pkgs, ... }:
+{
+  # srvos
+
+  environment.systemPackages = map lib.lowPrio [
```

Comment:
Could `emacs` (or `emacs-nox`) be added?

Reply:
Sure, I've added it to systemPackages in modules/shared/community-builder.nix.